src/app/(main)/my-voyage/[teamId]/tech-stack/components/TechStackCard.tsx

```diff
@@ -125,20 +125,24 @@ export default function TechStackCard({ title, data }: TechStackCardProps) {
                       ))}
                     </AvatarGroup>
                   </div>
-                  {element.teamTechStackItemVotes
-                    .map((item) => item.votedBy.member.id)
-                    .includes(userId) ? (
-                    <RemoveVoteBtn
-                      id={element.id}
-                      openMenu={setOpenMenuId}
-                      numberOfVotes={element.teamTechStackItemVotes.length}
-                      closeMenu={handleSettingsMenuClose}
-                      setIsEditing={setIsEditing}
-                      isMenuOpen={openMenuId}
-                    />
-                  ) : (
-                    <AddVoteBtn />
-                  )}
+                  {
```

Comment:
@JaneMoroz  That should be okay.

Reply:
@JaneMoroz wouldn't it be better to just do a ternary instead of 2 &&s? so like

```
{voteIsSubmitted ? (
      <RemoveVoteBtn
         id={element.id}
         openMenu={setOpenMenuId}
         numberOfVotes={element.teamTechStackItemVotes.length}
         closeMenu={handleSettingsMenuClose}
         setIsEditing={setIsEditing}
         isMenuOpen={openMenuId}
      />
 ): (
 <AddVoteBtn />
)}
```

Reply:
> @JaneMoroz wouldn't it be better to just do a ternary instead of 2 &&s? so like

Yeah, let's do it this way. For some reason I thought that a ternary would give the same error, but it works just fine, no errors